.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible.
+-->
+
+
+#### What happened:
+
+#### What you expected to happen:
+
+#### How to reproduce it (as minimally and precisely as possible):
+
+#### Anything else we need to know?:
+
+#### Environment:
```

Comment:
what about APIClarity version(we might need to integrate the version and commit-sha in the build phase to be able to extract it easily)? APIClarity logs?

Reply:
Didn't put the version, as we don't have versions yet. Sure, adding the logs